File: 프로그래머스/lv0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.java
import java.util.*;
class Solution {
 public int solution(int[] numbers) {
       int answer = 0;
        int[] ints = Arrays.stream(numbers).sorted().toArray();
        if(ints[0] * ints[1] < ints[ints.length-2] * ints[ints.length-1]) {
            return ints[ints.length-2] * ints[ints.length-1];
        } else return ints[0] * ints[1];
    }
}

File: 프로그래머스/lv0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.cs
using System;

public class Solution {
             public int solution(int[] numbers)
            {
              int max = int.MinValue;
                for (int i = 0; i < numbers.Length; i++)
                {

                    for (int j = 0; j < numbers.Length; j++)
                    {
                        if (i != j)
                        {
                            int sum = numbers[i] * numbers[j];
                            if (sum > max)
                            {
                                max = sum;
                            }
                        }
                    }
                }
                return max;
            }
}